data/root/root.file_list

```diff
@@ -291,9 +291,12 @@ ruby:
   # avoid update-alternatives
   r /etc/alternatives
   r /usr/bin/rake*
+  r /usr/bin/{y2,}racc*
   r /usr/bin/rdoc*
   r /usr/bin/ri*
   r /usr/bin/bundle*
+  /usr/lib64/ruby/*/{racc,rdoc}
+  /usr/lib64/ruby/gems/*/gems/{minitest,racc,rake,rdoc,test-unit}-*
```

Comment:
btw do you remove also gem cache in "/usr/lib64/ruby/gems/*/cache"? it can save some space.  maybe also "/usr/lib64/ruby/gems/*/doc" makes sense if not already removed

Reply:
gem cache is removed entirely later in the file

Reply:
good